modules/gathering-clusterversion-history.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-clusterversion-history_{context}"]
+= Gathering ClusterVersion history 
+
+The cluster version Operator (CVO) records updates made to a cluster, known as the ClusterVersion history. The entries can reveal correlation between changes in cluster behavior with potential triggers, although correlation does not imply causation. 
+
+[NOTE]
+====
+The ClusterVersion history has a size limit, so in cases where enough updates occurred to have reached that limit, the least relevant entries in the history are pruned to accommodate the limit. 
```

Comment:
hi @obrown1205 Shall we put more details about the limitation in the doc? like below in related bug, at least put the limit size 100 here:
```
Initial version is kept
all minor version updates are kept
the latest z streams updates are kept
however, the oldest z stream updates (in previous minor versions), which are not considered as important to the overall history of the cluster, are deleted from the history
```

Reply:
can do :)

Reply:
@LalatenduMohanty what do you think of this?

The initial, minor, and z-stream version updates are stored by the ClusterVersion history. However, the ClusterVersion history has a size limit, so in cases where enough updates occurred to have reached that limit, the oldest z stream updates in previous minor versions are pruned to accommodate the limit.

Reply:
This looks mostly correct . But I do not think we need to go for writing the details. Anyone who will go through history should know that we might have modified the history if the number of entries are 100.